### ISSUE TITLE
activating a timepicker when a dialog is opened

### ISSUE THREAD

**Issue by mosheliv** [joanpablo/reactive_forms#243]
I have a reactive_form dialog with a start_time and end_time timepicker. I want the start_time to be opened as soon as the dialog is opened, without the user having to tap the icon. Is that possible?

**Comment by vasilich6107** [comment]
@mosheliv it does not seem to be related to the form functionality.
Look at your DateTime picker control if it supports programmatically opening the picker and open it after first frame with registerPostframeCallback

**Comment by mosheliv** [comment]
You are right if the form does not supports it as a functionality, which I guess it does not. 
Thank you for helping even though it's unrelated, much appreciated.
Some things are just difficult in flutter.....

**Comment by vasilich6107** [comment]
@mosheliv one more time - your case does not seem to be the field of form responsibility 
This seems like responsibility of the particular control which you are using.
What type of DateTime picker are you using?

**Comment by mosheliv** [comment]
I understand, i was just thanking you for mentioning the postframecallback which i wasn't aware of. Not sure it will help.
I am using the one used in the example reactivetimepicker

**Comment by vasilich6107** [comment]
@mosheliv could you attach a link to exact line of the example file?

**Comment by mosheliv** [comment]
https://github.com/joanpablo/reactive_forms/blob/47ba5c3d59f625bba191efdb9f0de7f552d8dec7/example/lib/samples/complex_sample.dart#L170

**Comment by vasilich6107** [comment]
@mosheliv it seems that this control does not support programatical opening(
It is the point of improvements

**Comment by ahmedabdelhafez** [comment]
can you post  your code how to create date picker or time picker

**Comment by mosheliv** [comment]
You can look at the demo, it has one
